staker/block_validator.go

```diff
@@ -585,7 +585,10 @@ func (v *BlockValidator) createNextValidationEntry(ctx context.Context) (bool, e
 		return false, fmt.Errorf("illegal batch msg count %d pos %d batch %d", v.nextCreateBatchMsgCount, pos, endGS.Batch)
 	}
 	var comm espressoTypes.Commitment
-	var isHotShotLive bool
+	isHotShotLive, err := v.lightClientReader.IsHotShotLive(msg.Message.Header.BlockNumber)
+	if err != nil {
+		return false, fmt.Errorf("error fetching the hotshot liveness")
```

Comment:
nit: I think we should also attach the error here like  - `fmt.Errorf("error fetching the hotshot liveness: %v", err)`

Reply:
also attach the block number